README.md

```diff
@@ -176,27 +175,70 @@ head .cache/v/pytest_aws:cloudservices-aws-stage:us-west-2:rds:describe_db_insta
 
 These files can be removed individually or all at once with [the pytest --cache-clear](https://docs.pytest.org/en/latest/cache.html#usage) option.
 
-### Test Severity
+## Test Config
 
-pytest-services adds the command line arg `--severity-config` for
-adding a severity marker to tests. A severity can be `INFO`, `WARN`,
-or `ERROR`.
+pytest-services add the optional `--config` cli option for passing in a custom config file.
+
+Example config in repo (`config.yaml.example`):
+```
+exemptions:
+  - test_name: test_ec2_instance_has_required_tags
+    test_param_id: i-0123456789f014c162
+    expiration_day: 2019-01-01
+    reason: ec2 instance has no owner
+  - test_name: test_ec2_security_group_opens_specific_ports_to_all
+    test_param_id: '*HoneyPot'
+    expiration_day: 2020-01-01
+    reason: purposefully insecure security group
+severities:
+  - test_name: test_ec2_instance_has_required_tags
+    severity: INFO
+  - test_name: '*'
+    severity: ERROR
+regressions:
+  - test_name: test_ec2_security_group_opens_all_ports_to_all
+    test_param_id: '*mycustomgroup'
+    comment: this was remediated by ops team
+aws:
+  required_tags:
+    - Name
+    - Type
+    - App
+    - Env
+  whitelisted_ports_global:
+    - 25
+  whitelisted_ports:
+    - test_param_id: '*bastion'
+      ports:
+        - 22
+        - 2222
+```
+
+The `--aws-require-tag` and `--aws-whitelisted-ports` will overwrite whatever is present in the config file.
```

Comment:
We could just nuke these options too if you want to put everything in the config file.